generators/generator-multibot/generators/app/templates/settings/appsettings.json

```diff
@@ -1,30 +1,35 @@
 {
-  "feature": {
-    "UseShowTypingMiddleware": false,
-    "UseInspectionMiddleware": false,
-    "RemoveRecipientMention": false
-  },
+  "MicrosoftAppPassword": "",
```

Comment:
Don't think this is something we need to change in this PR, but I think seeing these changes is telling me we should find a way to centralize as much of this as possible into the generator-microsoft-bot-adaptive layer. I'll follow up on this later and see what we can do.